features/org.wso2.carbon.appmgt.store.feature/src/main/resources/store/extensions/assets/webapp/pages/myapps.jag

```diff
@@ -149,17 +149,52 @@ require('/modules/store.js').exec(function (ctx) {
             options.lifecycleState = ["published"];
             assets = store.taggeds(type, options, paging);
         } else if (query) { // if search from search box
-            urlQuery = 'query=' + query + '&';
-            query = require('/modules/search.js').build(query);
-            query.overview_treatAsASite = 'FALSE';
-            if (!isMultipleVersionDisplayEnabled) {
-                query.overview_makeAsDefaultVersion = 'true';
+            var searchParameter;
+            var searchValue;
+            var splitQuery = query.split(":");
+            searchParameter = splitQuery[0];
+            // This is a registry based search. In search box, user wants to search by business owner name. But
+            // in registry, there is only business owner Id. Therefore, it is necessary to get business owner Ids
+            // related to search prefix from DB.
+            if (!searchParameter.equals("Business Owner")) {
+                urlQuery = 'query=' + query + '&';
+                query = require('/modules/search.js').build(query);
+                query.overview_treatAsASite = 'FALSE';
+                if (!isMultipleVersionDisplayEnabled) {
```

Comment:
This logic is used to search webapps by business owner in store. So, we need to add overview_treatAsASite = 'FALSE' to urlQuery to get webapp only. There is a different search query for sites as well. 
overview_makeAsDefaultVersion = 'true'  is also use to make the search query. Search in Store is based on registry search. So we have add all the field types that want to do the correct search.